frontend/src/app/workflows/workflows.component.ts

```diff
@@ -0,0 +1,163 @@
+import {HttpClient} from '@angular/common/http';
+import {Component, HostListener, NgZone, OnInit, ViewChild} from '@angular/core';
+
+import {ToastService} from 'ng-bootstrap-ext';
+import {EditorConfiguration} from 'codemirror';
+import {PrivacyService} from '../privacy.service';
+import {LintService} from '../shared/lint.service';
+import {WorkflowsService} from './workflows.service';
+import {GenerateResult} from './model/GenerateResult';
+import {IOutputData, SplitComponent} from 'angular-split';
+import {environment} from '../../environments/environment';
+import {cmWorkflowsHint} from './model/helper/workflows-hint';
+
+@Component({
+  selector: 'app-workflows',
+  templateUrl: './workflows.component.html',
+  styleUrls: ['./workflows.component.scss']
+})
+export class WorkflowsComponent implements OnInit {
+  @ViewChild('split') split!: SplitComponent;
+
+  public content!: any;
```

Comment:
Please no any

Reply:
Also, you use `public` a lot. Since that is the default visibility I find it kind of unnecessary and it adds clutter. Could be removed everywhere.

Reply:
done